src/components/WindowTopBar.js

```diff
@@ -29,7 +29,7 @@ export class WindowTopBar extends Component {
     } = this.props;
     return (
       <AppBar position="relative">
-        <Toolbar disableGutters className={classNames(classes.windowTopBarStyle, focused ? classes.focused : null, ns('window-top-bar'))} variant="dense">
```

Comment:
I think the idiomatic way is to do some trickery in the styles:

```
  focused: {},
  ....
  windowTopBarStyle: {
    '&$focused': {
      borderTop: `2px solid ${theme.palette.secondary.main}`,
    },
```

(e.g. https://github.com/mui-org/material-ui/blob/4a4650572c037190ef44e290c7f3bcb93c32c1b1/packages/material-ui/src/ExpansionPanelSummary/ExpansionPanelSummary.js#L25)

Reply:
![makes total sense](https://media.tenor.com/images/585feadfd7ee5c4262166ca10e6492ea/tenor.gif)